app/src/main/res/layout/preference_search_add_engine.xml

```diff
@@ -10,14 +10,14 @@
     android:layout_width="match_parent"
     android:background="?android:selectableItemBackground"
     android:clickable="true"
-    android:focusable="true">
+    android:focusable="true"
+    android:layout_marginStart="@dimen/top_bar_alignment_margin_start">
```

Comment:
If we switch from margin to padding, the ripple background should be expand the full width of the row.

```suggestion
    android:paddingStart="@dimen/top_bar_alignment_margin_start"
    tools:ignore="RtlSymmetry">
```